automated-demo/auto-demo.sh

```diff
@@ -244,6 +244,7 @@ setup_kubefed()
 {
   download_kubefed_binary
   echo "Creating namespace ${DEMO_NAMESPACE} for deploying the demo"
+  run_ok_or_fail "oc --context=feddemocl1 create ns kube-federation-system" "0" "1"
```

Comment:
We should add a check to ensure this namespace doesn't already exist. If it exists, we should check if the operator is already running $KUBEFED_RELEASE.